gcloud-java-bigquery/src/test/java/com/google/gcloud/bigquery/testing/RemoteBigQueryHelperTest.java

```diff
@@ -14,12 +14,14 @@
  * limitations under the License.
  */
 
-package com.google.gcloud.bigquery;
+package com.google.gcloud.bigquery.testing;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
+import com.google.gcloud.bigquery.BigQuery;
 import com.google.gcloud.bigquery.BigQuery.DatasetDeleteOption;
+import com.google.gcloud.bigquery.BigQueryOptions;
 import com.google.gcloud.bigquery.testing.RemoteBigQueryHelper;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [No need to import a type that lives in the same package](https://www.codacy.com/app/mziccard/gcloud-java/file/2386255367/issues/source?bid=3123103&fileBranchId=3228105#l25)